src/sagas/index.js

```diff
@@ -183,6 +184,16 @@ export function* fetchByPage () {
   }))
 }
 
+export function* fetchConsoles ({ payload: { vmId } }) {
```

Comment:
Exporting this function seems not necessary. Consider also removing `return` as this saga is not called directly.